fhir-server/liberty-config/config/default/fhir-server-config.json

```diff
@@ -54,6 +54,12 @@
                     "max.block.ms": "60000",
                     "max.in.flight.requests.per.connection": "5"
                 }
+            },
+            "nats": {
+                "enabled": true,
+                "cluster": "nats-streaming",
+                "channel": "fhirNotifications",
+                "server": "172.30.255.3:4222,172.30.255.4:4222"
```

Comment:
what are these IPs?

Reply:
These are the nats streaming cluster IPs for the docker-compose implementation we are working on.  The kafka extension uses "localhost:9093" & in a final submission, these would be generalized to something like "localhost:4222".  Looking for the right balance of what to config in the json vs what to configure via a subscription resource.